packages/settingeditor/style/base.css

```diff
@@ -320,18 +277,34 @@
   height: 100%;
 }
 
+.jp-SettingsForm .jp-Buttonbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  margin: 8px;
+  gap: 0.5em;
+  max-width: 800px;
```

Comment:
800px is a very arbitrary threshold, putting the button in a bit weird place on larger screens:

![Screenshot from 2023-04-18 20-30-16](https://user-images.githubusercontent.com/5832902/232885140-12c89814-9f66-4511-8fac-f0f96827aea7.png)

Why not right-align it?